app/controllers/idv/capture_doc_status_controller.rb

```diff
@@ -85,7 +85,7 @@ def had_barcode_attention_result?
         idv_session.had_barcode_attention_error = session_result.attention_with_barcode?
       end
 
-      flow_session[:had_barcode_attention_error]
+      idv_session.had_barcode_attention_error
```

Comment:
I checked for remaining uses of flow_session[:had_barcode_attention_error] and I see several in [capture_doc_status_controller_spec.rb](https://github.com/18F/identity-idp/blob/234e7fc5b62848eee5670545fe16733383b90265/spec/controllers/idv/capture_doc_status_controller_spec.rb#L151) that could be shifted over to use idv_session.

Reply:
I left those in the specs because it is testing both, e.g.

```ruby
it 'assigns flow session values as having received attention result' do
  get :show

  expect(flow_session[:had_barcode_attention_error]).to eq(true)
  expect(controller.user_session[:idv][:had_barcode_attention_error]).to eq(true)
end
```